### DISCUSSION TITLE
Ability to use variables in transformations

### DISCUSSION THREAD

**Discussion by gingerwizard** [grafana/grafana#40924] [Platform]
It would be useful to be able to utilise variables in transforms -= specifically the "Add field from calculation" transform. This includes the [global variables](https://grafana.com/docs/grafana/latest/variables/variable-types/global-variables/).

Example usage - need to calculate a rate (e.g. per min) based using the `$**from and $**to` and a computed field. Assuming the underlying data source doesn't support.

**Comment by gingerwizard** [comment]
Existing issue https://github.com/grafana/grafana/issues/25469

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
